html/semantics/embedded-content/the-iframe-element/support/iframe-that-opens-modals.html

```diff
@@ -0,0 +1,30 @@
+<html>
+<body>
```

Comment:
The `<html>` and `<body>` are unnecessary (and the closing tags), but please add a `<!doctype html>`.

Reply:
Done